scripts/test-abnf-cts.rb

```diff
@@ -24,6 +24,13 @@
   "union array slice, underflowing to value with negative step",
   "union array slice, overflowing step",
   "union array slice, underflowing step",
+  "index selector, overflowing index",
+  "slice selector, overflowing to value",
+  "slice selector, underflowing from value",
+  "slice selector, overflowing from value with negative step",
+  "slice selector, underflowing to value with negative step",
+  "slice selector, overflowing step",
+  "slice selector, underflowing step",
 ]
```

Comment:
I'm not sure I follow what's going on here. Overflows/underflows should cause well formed expressions to fail parsing because they are invalid. Is the purpose of this Ruby file to exercise the ABNF alone or the spec more generally?

Reply:
@cabo I get it now - I didn't see that the above tests are skipped until now.

Reply:
Sorry for pushing this in the middle of your PR...  Opening up a new PR just for this seemed too much work.
The current tests indeed use the CTS (and JPC) only for checking the ABNF.